Controller/Process/Redirect.php

```diff
@@ -224,12 +224,7 @@ public function execute()
 
 						$this->_redirect('checkout/onepage/success', ['_query' => ['utm_nooverride' => '1']]);
 					} else {
-						$order->addStatusHistoryComment(__('3D-secure validation was unsuccessful.'))->save();
-
-						// Move the order from PAYMENT_REVIEW to NEW, so that can be cancelled
-						$order->setState(\Magento\Sales\Model\Order::STATE_NEW);
-						$this->_adyenHelper->cancelOrder($order);
-						$this->messageManager->addErrorMessage("3D-secure validation was unsuccessful");
+						$order->addStatusHistoryComment(__('3D-secure validation was unsuccessful. This order will be cancelled when the related notification has been processed.'))->save();
```

Comment:
I would quickly explain here why we don't cancel it immediately. So that there is a possibility that there is a successful authentication as well.

Reply:
@rikterbeek added a comment to explain it, if you think the wording is ok we can use it: https://github.com/Adyen/adyen-magento2/blob/d1e6d13c2c4abd584f5272e5cf704c68567dd368/Controller/Process/Redirect.php#L227-L231